lib/adapters/fs.js

```diff
@@ -14,7 +15,7 @@ module.exports = objectAssign({}, FS, {
 
   writeFile: fs.writeFile,
 
-  mkdir: fs.mkdir,
+  mkdir: mkdirp,
```

Comment:
I don't know exacly the advantages but I will need `fs-extra` to delete the created folders in the integration test. So I think I should also use it in the lib to not have one extra package to install ...?